src/clients/feed/feed.ts

```
@@ -67,12 +66,18 @@ class Feed {
     Returns a socket to listen for feed updates
   */
   listenForUpdates() {
-    if (!this.channelConnected) {
-      this.channel.join().receive("ok", () => {
-        this.channelConnected = true;
-      });
+    try {
+      this.channel.join();
 
       this.channel.on("new-message", (resp) => this.onNewMessageReceived(resp));
+    } catch (e) {
+      // Phoenix raises an error when trying to call join more than once. Given calling listenForUpdates unintentionally
+      // may happen fairly often, we choose to supress the error and print a warning.
+      if (e.message === "tried to join multiple times. 'join' can only be called a single time per channel instance") {
```

I think it makes sense to suppress this error only